lib/galaxy/tool_util/deps/conda_util.py

```diff
@@ -104,7 +104,7 @@ def __init__(
     ) -> None:
         self.condarc_override = condarc_override
         if not conda_exec and use_path_exec:
-            conda_exec = which("conda")
+            conda_exec = which("mamba")
```

Comment:
This won't do much as `use_path_exec` seems to always be False. You would need to change line 124 too.

Reply:
Done